browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -8746,19 +8746,19 @@ index 2e90534ffd8da83b7dc54d46fa7def16319bbb43..2493c00d58957751c65c37eb409fa8d6
      int innerLineHeight() const override;
  #endif
 diff --git a/Source/WebCore/rendering/ScrollAlignment.h b/Source/WebCore/rendering/ScrollAlignment.h
-index 694008e0451edc5770142a0a6d9eed52b04ded80..ec93869f9486bdf7bd3bb56478c62469d2fa58b6 100644
+index 01bb6ef9e22e7c595588ab8b312792a08533aead..db8e2b16d00c91ea1d2fec0a41a7e8c32d571e75 100644
 --- a/Source/WebCore/rendering/ScrollAlignment.h
 +++ b/Source/WebCore/rendering/ScrollAlignment.h
-@@ -78,6 +78,7 @@ struct ScrollAlignment {
-     Behavior m_rectVisible;
+@@ -81,6 +81,7 @@ struct ScrollAlignment {
      Behavior m_rectHidden;
      Behavior m_rectPartial;
+     bool m_enableLegacyHorizontalVisibilityThreshold { true };
```

Comment:
Looks like m_enableLegacyHorizontalVisibilityThreshold and m_disableMinThreshold that we add in our patch are doing the same thing. We should try removing m_disableMinThreshold (in separate PR is fine).

Reply:
Ok, removing `m_disableMinThreshold` required this extra change:

https://github.com/dpino/WebKit/commit/e42db6aa7c83298196cba2a585fb347ae771217c